Nabia/HW3/R_OSPool.submit

```diff
@@ -0,0 +1,27 @@
+# File Name: R.submit
++SingularityImage = "/cvmfs/singularity.opensciencegrid.org/opensciencegrid/osgvo-r:3.5.0"
+
+# Specify your executable and arguments
+executable = spill_calculation.r
+arguments = $(spill)
+
+# Tell HTCondor what to name log, standard error, and standard output("screen output") files
+log = log/spill$(Process).log
+output = output/spill$(Process).out
```

Comment:
@NabiaShahreen Please change the output and error file so that they end up in the *errout* folder as stated in HW3. Apart from that everything else looks good.

Reply:
I fixed this. I changed the R_OSPool.submit file and also updated the HW3 folder.